__fixtures__/verify-workspace/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "clean-workspace-3",
```

Comment:
I think it's bit confusing using "workspace" here, since workspaces are the children packages, not the root one. Maybe "clean-modular-project-3" would be better?